src/database/queries/ApprovedItem.ts

```diff
@@ -77,6 +100,17 @@ export async function getApprovedItems(
       // sending it along as the now-decoded `externalId` value to Prisma.
       decodeCursor: (cursor) =>
         JSON.parse(Buffer.from(cursor, 'base64').toString('ascii')),
+      // to conform to the custom, non-prisma type, we need to explicitly define
+      // the structure of each node.
+      recordToEdge: (record) => {
+        return {
+          // typescript is making this more difficult than it needs to be. authors
+          // is returned based on `baseArgs` above, but because this property isn't
+          // in the prisma ApprovedItem type, we can't just do `record.authors` -
+          // we first have to cast it as `any`.
+          node: { ...record, authors: (record as any).authors },
+        };
+      },
```

Comment:
I remember when I was researching the solution for this, it was annoying. Prisma doesn't return the referenced Model/Type on a given Model/Type and once you've have that figured out, Typescript gets unhappy with something 🙃 

Question: Can we try to typecast it as the `ApprovedItem` type from `/database/types` instead of `any`? Or is Typescript not happy with it 😅

Reply:
this is a good suggestion. trying now. (have to push a commit here as my local instance is now refusing to connect to the databse. whee!)

Reply:
it worked!